.github/workflows/ci.yml

```diff
@@ -7,7 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

Comment:
This should probably be in the earthfile via `GIT CLONE` but I started with this so it's here for now.

Reply:
Removing this means we can remove the ignore file changes too